# Annotation resources and example data for RareVariantVis 2.0.1 package

Second version of RareVariantVis package aims to annotate, filter and present
        genomic single nucleotide and structural variants (especially rare ones) in a global, per
        chromosome way. It accepts variants directly from caller - for example GATK or Speedseq. Output of package are lists of variants together with adequate visualization. Visualization of variants is performed in two ways -
        standard that outputs png figures and interactive that uses
        JavaScript d3 package. Interactive visualization allows to
        analyze trio/family data, for example in search for causative
        variants in rare Mendelian diseases. The package includes homozygous region caller and allows to analyse whole human genomes in less than 30 minutes. Here we present resources used by the package and explain how they were generated.


All_20160601_chr19.vcf.gz
File is 


CentromeresHg19.txt

CoriellIndex_S1.sv.vcf.gz

CoriellIndex_S1.vcf.gz

ExAC.r0.3.1.sites.vep_chr19.vcf.gz

RareVariants_CoriellIndex_S1.txt

RareVariants_Coriell_S2.txt

RareVariants_Coriell_S3.txt

StructuralVariants_CoriellIndex_S1.txt

StructuralVariants_Coriell_S2.txt

StructuralVariants_Coriell_S3.txt

UCSC_hg19_refSeq_160702.txt

nexterarapidcapture_exome_targetedregions_v1.2.bed

uniprot-all.txt
